vivisect/renderers/__init__.py

```diff
@@ -238,8 +239,10 @@ def renderLocation(self, mcanv, loc, name, isfunc, cmnt, extra):
 
             mcanv.addText(linepre, vatag)
             offset,bytes = self.vw.getByteDef(lva)
-            b = bytes[offset].encode('hex')
-            mcanv.addNameText(b, typename="undefined")
+            b = bytes[offset]
+            mcanv.addNameText(b.encode('hex'), typename="undefined")
+            if b in string.printable:
```

Comment:
Couple things ( aesthetic, i dig the idea ): 1. Not a huge fan of the two ; chars which will occur if a va is both undefined *and* has a real comment. 2. Lets stick to using spaces for display elements to cut down on compatibility issues. :)